appveyor.yml

```diff
@@ -121,8 +121,10 @@ test_script:
      --timeout 60
      --n-retries 2
      test\command\suite
+  # On Appveyor, インストール directory is recognized unicode escape sequence.
+  - set GROONGA_INSTALL_FOLDER_ESCAPED=\u7E67\uFF64\u7E5D\uFF73\u7E67\uFF79\u7E5D\u533B\u30FB\u7E5D\uFF6B
   - ruby test\command_line\run-test.rb
-     --groonga-install-prefix=%APPVEYOR_BUILD_FOLDER%\\\u7E67\uFF64\u7E5D\uFF73\u7E67\uFF79\u7E5D\u533B\u30FB\u7E5D\uFF6B\\%GROONGA_INSTALL_FOLDER%
+     --groonga-install-prefix=%APPVEYOR_BUILD_FOLDER%\\%GROONGA_INSTALL_FOLDER_ESCAPED%\\%GROONGA_INSTALL_FOLDER%
```

Comment:
I think that `GROONGA_INSTALL_FOLDER_ESCAPED` and `GROONGA_INSTALL_FOLDER` are easily confused names.

Reply:
Oh, I've also misunderstood it. :sob: